main.cpp

```diff
@@ -75,378 +74,232 @@ send me a DM to check your pull request
 /*
  1)
  */
-struct GymPlan
-{
-    bool member = true;
-    int daysPerWeek = 4;
-    
-    unsigned int numMembers = 9;
 
-    struct Membership
-    {
-        
-        void enterPerson()
-        {
+ /*
+GymPlan:
+    Has list of people signed up
+    can add people to list
+    can remove people from list
+ */
 
-        }
-        void declinePerson()
-        {
+#include <vector>
 
-        }
-    };
-    
-    void goToGym(bool member, int daysPerWeek);
+struct GymPlan
+{
+    int daysPerWeek = 4;
+    bool canAccessPool = true;
+    bool canBringGuests = true;
+    bool canAccessSpa = true;
     
-    Membership enter, decline;
+    std::vector<Person> members; //Has list of people signed up
+    void addMember(Person p);       //can add people to list
+    void removeMember(Person p);    //can remove people from list
 };
 
-void GymPlan::goToGym(bool, int)
+void GymPlan::addMember(Person)
 {
-    if(member == true)
-    {
-        enter.enterPerson();
-    }
-    else
-    {
-        decline.declinePerson();
-    }
+   
+}
+void GymPlan::removeMember(Person)
+{
+    
 }
+
 /*
  2)
- */
-struct DogWalkerPlan
-{
-    int daysPerWeek = 3;
-    float milesPerWalk = 2.5;
-    bool dogClean = true;
-    unsigned int timesWalked = 22;
 
-    struct Dog
-    {
-        void walkDog()
-        {
+times walked per week
+Notifies walker to start dog walk
 
-        }
+can either jog with the dogs or just walk them
+has a list of dogs to walk
+option to feed dog
 
-        void washDog()
-        {
+ 
+ */
 
-        }
-    };
-    void exerciseDog(int breed, bool dogClean);
+struct Dog
+{
+    bool needsFeeding = false;
+};
 
-    Dog walk, wash;
+
+struct DogWalkerPlan
+{
+    //times walked per week
+    int timesWalkedWeekly = 1;
+    //can either jog with the dogs or just walk them
+    bool involvesJogging = true;
+    
+    std::vector<Dog> dogsToWalk; //has a list of dogs to walk
+    
+    //Notifies walker to start dog walk
+    void notifyWalker();
+    
+    //option to feed dog
+    bool shouldFeedDogs = true;
+    void feedDog(int whichDogToFeed);
+    void feedAllDogs();
 };
-void DogWalkerPlan::exerciseDog(int, bool)
+void DogWalkerPlan::feedDog(int)
 {
-    if(dogClean == true)
-    {
-        walk.walkDog();
-    }
-    else 
-    {
-        wash.washDog();
-    }
+
+}
+void DogWalkerPlan::feedAllDogs()
+{
+
 }
 /*
  3)
+3 things it can do:
+    1) play
+    2) stop
+    3) sync tempo
+
+5 properties:
+    1)is edm
+    2)is rock
+    3)is hiphop
```

Comment:
You could summarize these 3 with a single property:  `int musicType`
- [ ] consolidate your types. 
- [ ] add 2 other properties.  maybe `volumeLevel` or `setLengthInMinutes`